### ISSUE TITLE
[Feature Idea] -  Setting for time before switching editing/preview 

### ISSUE THREAD

**Issue by BrunoGomesCoelho** [OwnJoke/obsidian-budget-wysiwyg#7]
Thanks for the great plugin!

Small feature idea, one thing that I feel would really help would be to have [the time between switching to preview](https://github.com/OwnJoke/obsidian-budget-wysiwyg/blob/31bae6745a0ba8bb51972ee5feb4e96f360cb435/main.ts#L62) user configurable - I currently find the 2 seconds too quick.

**Comment by OwnJoke** [comment]
Thanks for the good suggestion, this will probably be implemented eventually but I am currently on vacation for the next few weeks with no way to code.

**Comment by DutchPete** [comment]
@BrunoGomesCoelho after installing it I soon got annoyed with the quicks change to Preview, so I disabled the plug-in.

I was going to make the same request, then saw yours. I opened the `main.js` file and changed the 2000 to 4000 (4000 milliseconds = 4 seconds); the change to Preview was slower indeed.

**Comment by 27escape** [comment]
Thanks for that suggestion @DutchPete, I have done the same to my install, changed to 10000 (10s), so having this as a editable config field would be a great addition

**Comment by DutchPete** [comment]
@27escape yes, my 4000 was still not enough, so I changed to 6000, then 8000. It still was somewhat annoying, but I got fed-up and disabled the plug-in, albeit with regret.

I have set it to 10000 and will try it again. If I would need to make it any slower I think it would defeat the objective.

**Comment by slaughtr** [comment]
I have addressed this issue in #12

**Comment by DutchPete** [comment]
I stopped using the plug-in because even at 10,000 it was not working well. Now using Live Preview.

**Comment by OwnJoke** [comment]
yeah the plugin is currently broken and I was planning on fixing it this weekend but things came up. Ill try and fix it during the week, and once live preview comes out for everyone this plugin will be pretty much useless.
